src/pages/Welcome/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from 'react-router-dom';
+import {
+  Button, Logo, Subtitle, Title,
+} from '../../components/common';
+import { RouterPaths } from '../../constants';
+
+function Welcome() {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <div>
+        <Logo />
+        <Title text="Welcome to Target" />
```

Comment:
Is this kind of component necessary? I would have used the `@apply` rule in css to apply the classes to the different headings and then in the code just use `h1`, `h2`, etc. How would you call the components for `h3`, `h4`, `h5`, and `h6`? I think it is better if the level of the heading is clear so it is easier to spot if heading levels are being skipped or misused for accessibility